script/osht.sh

```diff
@@ -0,0 +1,470 @@
+###############################################################################
```

Comment:
This is a 3rd party file. When bundling dependencies it makes sense to provide a link to the original source and some versioning information. Since the abbreviation "osht" is meaningless to me it would also be nice to have a short description what this 3rd party script actually does.

Reply:
The test in circleci needs privileged container, thus must be run with 'machine' executor, which is Debian stretch. So I think the easiest way will be just downloading needed file from github at the moment.

Reply:
yep, you do that already and that is fine.